<launch>

  <node name="set_pick_pose" pkg="panda_pick_place" type="set_pick_pose.py" respawn="false" output="screen">
  </node>

</launch>
